packages/datafile-manager/__test__/httpPollingDatafileManager.spec.ts

```diff
@@ -134,28 +134,26 @@ describe('httpPollingDatafileManager', () => {
       manager.start();
       expect(manager.responsePromises.length).toBe(1);
       await manager.responsePromises[0];
-      expect(manager.get()).toEqual({ foo: 'bar' });
+      expect(JSON.parse(manager.get())).toEqual({ foo: 'bar' });
       updateFn.mockReset();
 
       await advanceTimersByTime(300000);
 
       expect(manager.responsePromises.length).toBe(2);
       await manager.responsePromises[1];
       expect(updateFn).toBeCalledTimes(1);
-      expect(updateFn).toBeCalledWith({
-        datafile: { fooz: 'barz' },
-      });
-      expect(manager.get()).toEqual({ fooz: 'barz' });
+      expect(updateFn.mock.calls[0][0]).toEqual({ datafile: '{"fooz": "barz"}' });
```

Comment:
Suggest this instead:
```suggestion
      expect(updateFn).toBeCalledWith({
        datafile: '{"fooz": "barz"}',
      });
```